app/renderer/client-frameworks/framework.js

```diff
@@ -0,0 +1,93 @@
+export default class Framework {
+
```

Comment:
Could we put client-frameworks and highlighting directories into a utils or helpers directory?